src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -126,30 +126,20 @@ class DashboardScreen extends Component<Props, State> {
 
   setQuery = (query: string) => this.setState({ query });
 
-  scrollToTransactionList = () => {
-    // hack, there is no scrollTo method available on SectionList, https://github.com/facebook/react-native/issues/13151
+  scrollTo = (offset: number) => {
     // @ts-ignore
```

Comment:
keep the comment above ts-ignore, please
// hack, there is no scrollTo method available on SectionList, https://github.com/facebook/react-native/issues/13151